app/src/main/java/com/framgia/englishconversation/screen/videoDetail/VideoDetailViewModel.java

```diff
@@ -56,13 +58,20 @@ public void onItemUserNameClick(TimelineModel item) {
                             .equals(item.getCreatedUser().getId())) {
                         return;
                     }
-                    mContext.startActivity(
-                            ProfileUserActivity.getInstance(mContext, item.getCreatedUser()));
+                    mContext.startActivity(ProfileUserActivity.getInstance(
+                            mContext,
+                            item.getCreatedUser()));
                 }
+
+                @Override
```

Comment:
ERROR: 'method def modifier' have incorrect indentation level 16, expected level should be one of the following: 8, 12.